alloy/src/main/java/com/liferay/faces/alloy/render/internal/AlloyRendererBase.java

```diff
@@ -44,19 +46,19 @@ public void decodeClientBehaviors(FacesContext facesContext, UIComponent uiCompo
 	@Override
 	public void encodeBoolean(ResponseWriter responseWriter, String attributeName, Boolean attributeValue,
```

Comment:
@stiemannkj1 Feel free to rename it to encodeBooleanProperty after you have updated the generator.